selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/utils/BrowserLogsUtil.java

```diff
@@ -25,27 +27,115 @@
+  }
+
+  /** filter data and get requests/responses that has been sent on CHE /api/ URL */
+  public static String getNetworkDataSentOnCheApi(WebDriver seleniumWebDriver) {
+    StringBuilder data = new StringBuilder("Network logs: \n").append("---------------\n");
+    JsonParser jsonParser = new JsonParser();
+    getPerformanceLogs(seleniumWebDriver)
+        .forEach(
+            logEntry -> {
+              JsonElement jsonElement = jsonParser.parse(logEntry.getMessage());
+              JsonObject jsonMessageNode =
+                  jsonElement.getAsJsonObject().get("message").getAsJsonObject();
+              String networkValue = jsonMessageNode.get("method").getAsString();
+
+              if (networkValue.equals("Network.requestWillBeSent")) {
+                data.append(getRequestsSentOnChe(jsonMessageNode));
+
+              } else if (networkValue.equals("Network.responseReceived")) {
+                data.append(getResponsesSentOnChe(jsonMessageNode));
+              }
+            });
+    return data.toString();
+  }
+
+  /**
+   * check that current request contains invocation to СHE api URL and provide information about URL
+   * and http method/status
+   *
+   * @param requestMessage json representation of the message object from the log
+   * @return info about request from the WebDriver
+   */
+  private static String getRequestsSentOnChe(JsonObject requestMessage) {
+    JsonObject requestNode = requestMessage.getAsJsonObject("params").getAsJsonObject("request");
+    StringBuilder requestInfo = new StringBuilder();
+    if (isLogEntryContainsApiUrl(requestNode)) {
+      requestInfo
+          .append("Request Info :---------------> \n")
+          .append("Method: " + requestNode.get("method"))
+          .append("\n")
+          .append("URL: " + requestNode.get("url"))
+          .append("\n\n");
+    }
+    return requestInfo.toString();
+  }
+
+  /**
+   * check that current response contains invocation to СHE api URL and provide information about
+   * URL and http method/status
+   *
+   * @param requestMessage json representation of the message object from the log
+   * @return info about request from the WebDriver
+   */
+  private static String getResponsesSentOnChe(JsonObject requestMessage) {
+    JsonObject responseNode = requestMessage.getAsJsonObject("params").getAsJsonObject("response");
+    StringBuilder responceInfo = new StringBuilder();
+    if (isLogEntryContainsApiUrl(responseNode)) {
+      responceInfo
+          .append("Response Info : <--------------- \n")
+          .append("Method: " + responseNode.get("status"))
+          .append("\n")
+          .append("URL: " + responseNode.get("url"))
+          .append("\n\n");
+    }
+    return responceInfo.toString();
+  }
+
+  private static boolean isLogEntryContainsApiUrl(JsonObject node) {
+    return (node.get("url").isJsonNull()) ? false : node.get("url").getAsString().contains("/api/");
```

Comment:
there could be requests to other resources with "/api/" in URL. It's more reliable to check on Che hostname as well.